config/dev/gcp-clusterdeployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: ClusterDeployment
+metadata:
+  name: gcp-${CLUSTER_NAME_SUFFIX}
+  namespace: ${NAMESPACE}
+spec:
+  template: gcp-standalone-cp-0-1-0
+  credential: gcp-credential
+  config:
+    clusterLabels: {}
+    clusterAnnotations: {}
+    project: k0rdent-dev
+    region: us-east4
+    network:
+      name: default
+    controlPlane:
+      instanceType: n1-standard-2
+      image: projects/ubuntu-os-cloud/global/images/ubuntu-2004-focal-v20250213
```

Comment:
out of curiosity: how does this path get to FQDN? Does it get a prefix by the corresponding CAPI controller?